gui/src/components/onboarding/pages/body-proportions/ProportionsChoose.tsx

```diff
@@ -151,6 +154,13 @@ export function ProportionsChoose() {
     setImportState(ImportStatus.SUCCESS);
   };
 
+  const resetAll = () => {
+    sendRPCPacket(
+      RpcMessage.SkeletonResetAllRequest,
+      new SkeletonResetAllRequestT()
+    );
+  };
```

Comment:
This call returns a `RpcMessage.SkeletonConfigResponse`, which triggers saving exported body proportions.

Reply:
did u fix it?

Reply:
Nah idk the best solution to this

Reply:
i dont know the best one either but i will try i guess

Reply:
what about now?

Reply:
Yeah that's probably fine, a tiny bit cursed but meh